include/certifier_framework.h

```diff
@@ -109,40 +146,155 @@ namespace certifier {
       void delete_blob_by_index(int index);
       int get_num_blobs();
 
+      // Serialize contents of the policy store.
```

Comment:
How can we explain this better? I think this comment by itself is not very helpful ... .???